packages/scripts/src/dbt/generateModel.js

```diff
@@ -79,6 +81,24 @@ async function getColumnsInRelation(client, schemaName, tableName) {
   }));
 }
 
+/**
+ * @param {pg.Client} client
+ * @param {number} oid
+ * @returns {Array<{ oid: number; name: string; }>} A list of triggers that exist on the given table.
+ */
+async function getTriggers(client, oid) {
+  return (
+    await client.query(
+      `SELECT oid, tgname as name
+      FROM pg_trigger
+      WHERE tgisinternal = false AND tgrelid = $1`,
+      [oid],
+    )
+  ).rows
+    .filter(({ name }) => !name.startsWith('RI_'))
```

Comment:
What's the reason for filtering out the `RI_*` triggers? A comment might be nice :+1:

Reply:
They're postgres internals. idk why they're called RI_